Sunrise/Models/DiscountedLineItemPriceForQuantity.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright (c) 2016 Commercetools. All rights reserved.
+//
+
+import ObjectMapper
+
+struct DiscountedLineItemPrice: Mappable {
```

Comment:
Is it there a reason the file-name differs slightly from the struct it contains?
 `DiscountedLineItemPriceForQuantity.swift` vs. `DiscountedLineItemPrice` ?